diff -dur winpdb-1.4.6.orig/rpdb2.py winpdb-1.4.6/rpdb2.py
--- winpdb-1.4.6.orig/rpdb2.py	2009-04-01 20:32:35.000000000 +0200
+++ winpdb-1.4.6/rpdb2.py	2010-02-26 13:00:48.000000000 +0100
@@ -10593,8 +10593,9 @@
             b = ' --base64=%s' % _b
 
         debugger = os.path.abspath(__file__)
-        if debugger[-1:] == 'c':
-            debugger = debugger[:-1]
+        if debugger[-1:] in ('c', 'o'):
+            if os.path.exists(debugger[:-1]):
+                debugger = debugger[:-1]
 
         debugger = as_unicode(debugger, fse)
 
